### PR TITLE
hopefully adds "NT-Military Police"

### PR DESCRIPTION
**Pull request by thevandie**
## About The Pull Request

Adds NT Military Police as a title for sec-off

## Why It's Good For The Game

More variety is cool

## Changelog
:cl:
add: A new title for sec-off
/:cl:

### CODE REVIEW

**Comment by JRDead** [comment]
I'm a novice on this too, but I think it's well done. Good job on your first PR. :)

**Review by retlaw34 — CHANGES_REQUESTED** [review]
Please folow the PR template, please

**Comment by thevandie** [comment]
k I fixed it, hope it works

File: ModularBungalow/altjobtitles/alt_titles.dm [hunk 1]
```diff
@@ -100,7 +100,8 @@
 	senior_title = "Staff Sergeant"
 
 /datum/job/officer
-	alt_titles = list("Security Guard", "Deputy", "Hazardous Device Technician")
+
```
Comment (retlaw34) [inline ModularBungalow/altjobtitles/alt_titles.dm:103]:
```suggestion

```
Newline doesn't need to be here

**Review by retlaw34 — CHANGES_REQUESTED** [review]

**Comment by Kitsunemitsu** [comment]
Great first PR to learn how Github works!

**Review by BootlegBow — APPROVED** [review]
looks good

**Review by retlaw34 — APPROVED** [review]

**Pull request merged**